Python_Scripts/compareStateTrajectories_BioModels_2.py

```diff
@@ -3,50 +3,31 @@
 # => creates two important .tsv files
 
 # Attention:    boundary conditions are not being simulated by COPASI!
-# all trajectories simulated by COPASI were simulated with absolute and relative tolerances 1e-14 & 1e-14,
-# except three models where 1e-12 & 1e-12 had to be used.
+# all trajectories simulated by COPASI were simulated with absolute and
+#  relative tolerances 1e-14 & 1e-14, except three models where
+#  1e-12 & 1e-12 had to be used.
 
-from execute_loadModels import *
-from JWS_changeValues import *
-#from colourDataFrame import *
-import amici.plotting
 import numpy as np
-import matplotlib.pyplot as plt
-import libsbml
-import libsedml
-import time
-import statistics
 import pandas as pd
 import os
-import urllib.request
-import json
-import itertools
-import time
+
+from C import (
+    DIR_MODELS_TRAJ_AMICI, DIR_MODELS_AMICI, DIR_MODELS_BIOMODELS,
+    DIR_MODELS_TRAJ_REF)
 
 
 def compStaTraj_BioModels_2(MultistepMethod, atol, rtol):
     # upper and lower boundaries for the absolute and relative errors
     AbsError_1 = range(-20, 10)
     RelError_2 = range(-20, 10)
 
-    # important paths
-    base_path = '../../Benchmarking_of_numerical_ODE_integration_methods'
-    trajectory_path = base_path + '/biomodels_files/StateTrajectories_BioModels_COPASI_Data'
-    comparison_files_path = base_path + '/biomodels_files/' + MultistepMethod + '_' + atol + '_' + rtol
-    all_biomodels_path = base_path + '/BioModelsDatabase_models'
-
-    # create folder for all results
-    if not os.path.exists(comparison_files_path):
-        os.makedirs(comparison_files_path)
+    print(f"Setting: {MultistepMethod} {atol} {rtol}")
 
     # iterate over all error combinations
     for iAbsError in range(0, len(AbsError_1)):
         for iRelError in range(0, len(RelError_2)):
```

Comment:
Why don't we iterate over *one* quantity, if those things should be equal by construction?

Reply:
yes. was too lazy to change it, but trivially yes.